sceptre/scipool/config/bmgfki/sc-product-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -0,0 +1,72 @@
+template_path: "sc-product-ec2-linux-jumpcloud-notebook.j2"
```

Comment:
The name suggests that this product is accessed through JumpCloud.  Is that what we want for an S.C. instance set up for external users?